src/GraphQL/Types/FormType.php

```diff
@@ -35,7 +35,21 @@ public function fields(): array
             'rules' => [
                 'type' => GraphQL::type(ArrayType::NAME),
                 'resolve' => function ($form, $args, $context, $info) {
-                    return $form->blueprint()->fields()->validator()->rules();
+                    return collect($form->blueprint()->fields()->validator()->rules())
+                        ->map(function ($rules) {
+                            return collect($rules)->map(function ($rule) {
+                                if (is_string($rule)) {
+                                    return $rule;
+                                }
+
+                                if (method_exists($rule, '__toString')) {
```

Comment:
Although, not technically required, it might be nice to check for the [`Stringable`](https://www.php.net/manual/en/class.stringable.php) interface here, rather than the existence of a `__toString()` method?

```suggestion
                                if (method_exists($rule, '__toString')) {
                                if ($rule instanceof Stringable) {
```

Reply:
Yep, easily changed. Thats done.